### ISSUE TITLE
NApps are installed in old venv folder

### ISSUE THREAD

**Issue by cemsbr** [kytos/kytos-utils#85]
```
# load new_virtualenv1
rm ~/.kytosrc
kytos napps install kytos/of_core
# a new ~/.kytosrc will be created
# load new_virtualenv2
# delete new_virtualenv1
kytos napps install kytos/of_core
kytos napps list
# NApp is installed in new_virtualenv1 folder
# NApp new_virtualenv2 NApps folder is empty
# NApp won't be loaded by kytosd in new_virtualenv2
```

The problem is that `~/.kytosrc` file keeps old entries for napps dirs.

**Comment by renanrodrigo** [comment]
In a 'new' installation, `~/.kytosrc` is still the same, while (a new copy of) `/etc/kytos/kytos.conf` is installed on the detected `[BASE_ENV]`. The problem is both files contains a 'napps_dir' configuration.
`kytos-utils` uses the directory configured in `kytosrc` and `kytos` reads it from `kytos.conf`.

**Comment by beraldoleal** [comment]
This will be fixed after #86
